src/Build/Graph/ParallelWorkSet.cs

```diff
@@ -0,0 +1,196 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Build.Graph
```

Comment:
The namespace of the other graph related classes is `Microsoft.Build.Experimental.Graph`